charts/consul/templates/server-podsecuritypolicy.yaml

```diff
@@ -27,6 +27,15 @@ spec:
     - 'downwardAPI'
     - 'persistentVolumeClaim'
   hostNetwork: false
+  hostPorts:
+  {{- if .Values.server.exposeGossipAndRPCPorts }}
+  - min: 8300
+    max: 8300
+  - min: {{ .Values.server.ports.serflan.port }}
+    max: {{ .Values.server.ports.serflan.port }}
```

Comment:
Im not sure what happens here if the value of the `serflan.port` is greater than 8302. Does the next entry then clobber it?

Reply:
This definitely works with the default values. A touch worried about non-default ones.

Reply:
I don't think the next entry would clobber it. I think each array item is an independent port range.